recipe/meta.yaml

```diff
@@ -1,31 +1,32 @@
 {% set name = "fastapi" %}
-{% set version = "0.103.0" %}
+{% set version = "0.112.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4166732f5ddf61c33e9fa4664f73780872511e0598d4d5434b1816dc1e6d9421
+  sha256: b2537146f8c23389a7faa8b03d0bd38d4986e6983874557d95eed2acc46448ef
 
 build:
   number: 0
-  # httpx and its deps (httpcore, idna) aren't available on s390x. 
+  # httpx and its deps (httpcore, idna) aren't available on s390x.
   # httpx is required only for testing fastapi  by using `from fastapi.testclient import TestClient`
-  skip: true  # [py<37 or s390x]
+  skip: true  # [py<38 or s390x]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
     - hatchling
```

Comment:
```suggestion
```
The build backend is now pdm: https://github.com/fastapi/fastapi/blob/0.112.1/pyproject.toml#L2.